File: validation-service-demo/src/main/kotlin/de/senacor/innolab/graalvm/demo/validation/ValidationServiceDemoApplication.java
package de.senacor.innolab.graalvm.demo.validation;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(proxyBeanMethods = false)
public class ValidationServiceDemoApplication {

    public static void main(String[] args) {
        SpringApplication.run(ValidationServiceDemoApplication.class, args);
    }

}

File: validation-service-demo/src/main/kotlin/de/senacor/innolab/graalvm/demo/validation/ValidationServiceDemoApplication.kt
package de.senacor.innolab.graalvm.demo.validation

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class ValidationServiceDemoApplication

fun main(args: Array<String>) {
	runApplication<ValidationServiceDemoApplication>(*args)
}
